docs/usage.rst

```diff
@@ -234,6 +234,7 @@ downstream ``devenv.yml`` files will inherit that dependency.
 and `pip <https://pip.pypa.io/en/stable/user_guide/#constraints-files>`__ have the concept of *constraints*,
 which are package version restrictions that do not cause a package to be included in the environment directly,
 but if they are included in the environment by other means, then their version specifiers must be respected.
+However, for the current version dependencies declared in pip are ignored for constraints purposes.
```

Comment:
Let's make this more prominent, moving it next to the `.. note` at the bottom of this section.

```suggestion
```

I suggest to add it as a separate "note", like:



```rst

.. note::
    Currently dependencies via ``pip`` are ignored for constraints purposes.
```